src/backend/embedding/extract_file/extract_file.py

```diff
@@ -62,6 +65,8 @@ def extract_file(self, customer_guid: str, filename: str):
             file_type = self.file_extract.detect_file_type(local_path)
             if file_type == ".pdf":
                 logger.info(f"Verified file '{filename}' as a valid PDF.")
+            elif file_type == ".docx":
```

Comment:
Change .pdf and .docx as enums and declare it in class and use the ENUMs everywhere so that we dont hardcode these values everywhere